pyccel/codegen/printing/fcode.py

```diff
@@ -620,25 +626,25 @@ def _print_PythonPrint(self, expr):
                 if f.rank == 1:
                     print_body.append(space_end)
 
-                for_body = [PythonPrint(print_body)]
+                for_body = [PythonPrint(print_body, output_unit=expr.output_unit)]
                 for_loop = For(for_index, for_range, for_body, scope=loop_scope)
                 for_end_char = LiteralString(']')
                 for_end = FunctionCallArgument(for_end_char,
                                                keyword='end')
 
-                body = CodeBlock([PythonPrint([FunctionCallArgument(LiteralString('[')), empty_end]),
+                body = CodeBlock([PythonPrint([FunctionCallArgument(LiteralString('[')), empty_end], output_unit=expr.output_unit),
```

Comment:
I think @EmilyBourne will not be happy with this line because it's too long, as she said once if the line doesn't fits it screen you have to break the line.